### PR TITLE
Added CFML

### PR DESCRIPTION
**Pull request by petrichor8**

### CODE REVIEW

File: img/website/31.svg [hunk 1]
```diff
@@ -0,0 +1,318 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="250px" height="250px" viewBox="0 0 250 250" enable-background="new 0 0 250 250" xml:space="preserve">  <image id="image0" width="250" height="250" x="0" y="0"
```
Comment (Sambego) [inline img/website/31.svg:3]: Your svg image is a wrapped png file as base64. Can you add a real svg image?

File: views/website/libraries/31-CFML.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "CFML",
+  "uniqueClass": "cfml",
+  "image": "/img/31.svg",
+  "bgColor": "rgb(101,143,170)",
+  "libs": [
+    {
+      "minimumVersion": CF11,
```
Comment (Sambego) [inline views/website/libraries/31-CFML.json:8]: This should be a string

**Review by Sambego — CHANGES_REQUESTED** [review]

File: img/website/31.svg (hunk 1 above)
Comment (petrichor8) [reply img/website/31.svg]: should be good now!

**Comment by petrichor8** [comment]
updated as per requests

**Comment by Sambego** [comment]
I think you accidentally closed the pull request :)

**Review by Sambego — APPROVED** [review]

**Comment by petrichor8** [comment]
sorry, thought I had to in order for you to see the updates.  I'm just a dog clicking a mouse, no idea what I'm doing!  Thanks :)

**Comment by Sambego** [comment]
No worries, your changes are live on jwt.io 🎉

**Pull request merged**